### PR TITLE
Add run_exports

### PR DESCRIPTION
**Pull request by sdvillal**
Checklist
* [X] Used a fork of the feedstock to propose changes
* [X] Bumped the build number (if the version is unchanged)
* [X] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

See also https://github.com/conda-forge/ffmpeg-feedstock/pull/72

### CODE REVIEW

**Comment by sdvillal** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (jakirkham) [inline recipe/meta.yaml:23]: So we are ignoring [this exception]( https://abi-laboratory.pro/index.php?view=objects_report&l=lame&v1=3.97&v2=3.98.4 )? Probably reasonable as that was almost a decade ago and they have kept things a bit more stable since.

**Comment by jakirkham** [comment]
Should we pull packages not using `run_exports` once this is merged? If so, which packages would those be?

File: recipe/meta.yaml
Comment (sdvillal) [reply recipe/meta.yaml:23]: Isn't 3.97 -> 3.98 also captured by the x.x pin?

**Comment by sdvillal** [comment]
@jakirkham do you mean unpinned ffmpeg dependencies?

**Review by soumith — APPROVED** [review]
please let me know if i should merge this, I dont know if you have merge rights or not.

**Comment by sdvillal** [comment]
@soumith yes, please merge. Once this makes it to CF channel I will in turn take care of the ffmpeg PR.

**Pull request merged**
